changes/add_tests_for_dp5mirage.md

```diff
@@ -0,0 +1 @@
+Added tests for DP5Mirage Scanner
```

Comment:
Remove this file; the changelog is for user-facing and admin-facing changes. This PR doesn't need a changelog message.